core/src/main/resources/reference.conf

```diff
@@ -591,14 +591,14 @@ datastax-java-driver {
   profiles {
     akka-persistence-cassandra-profile {
       basic.request {
-        consistency = QUORUM
+        consistency = LOCAL_QUORUM
         # the journal does not use any counters or collections
         default-idempotence = true
       }
     }
     akka-persistence-cassandra-snapshot-profile {
       basic.request {
-        consistency = ONE
+        consistency = LOCAL_QUORUM
```

Comment:
I think it should be possible to use LOCAL_ONE for the reads but fails so might be required for the writes.